lib/widgets/chat.dart

```diff
@@ -103,110 +107,144 @@ class _ChatMessageState extends State<ChatMessage> {
     var isLast = widget.index == widget.msgs.length - 1;
     var isFirst = widget.index == 0;
 
-    return AnimatedOpacity(
-      curve: Curves.easeIn,
-      duration: Duration(milliseconds: 300),
-      opacity: opacity,
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Row(
-            mainAxisAlignment:
-                userSent ? MainAxisAlignment.end : MainAxisAlignment.start,
-            mainAxisSize: MainAxisSize.max,
-            crossAxisAlignment: CrossAxisAlignment.end,
-            children: [
-              if (!userSent)
-                Padding(
-                  padding: EdgeInsets.only(
-                    right: 14,
-                    left: 14,
+    return GestureDetector(
+      onTap: () {
+        setState(() {
+          isTimeSentVisible = true;
+        });
+      },
+      onLongPress: () {
```

Comment:
This should stop that behaviour, does it look good?

Reply:
Works, but I can't select text

Reply:
You have to long press and it will copy all of the text to the clipboard